_tools/riot-contributors.py

```diff
@@ -0,0 +1,34 @@
+import os
+import sys
+import json
+import requests
+
+CONTRIBUTORS_URL = "https://api.github.com/repos/RIOT-OS/RIOT/contributors"
+OBJ_PER_PAGE = 100
+MAX_PAGES = 20
+
+if __name__ == "__main__":
+    curr_page = 0
+    contributor_list = []
+    headers = {}
+    token = os.getenv('GITHUB_TOKEN')
+    if token:
+        headers["Authorization"] = "token {}".format(token)
```

Comment:
Without token the GitHub API rate limiting is far more restrictive: https://docs.github.com/en/rest/overview/resources-in-the-rest-api#rate-limiting. 50 requests per hour vs 5000 requests per hour. But sure, to make for a quick and easy build, building without a `GITHUB_TOKEN` set would be desirable.

Reply:
It should be possible to  just run the script without the token in the environment. You would just be affected by the constraints that @miri64 pointed out. This is documented in the README.

Reply:
yes, it works, but in this case the Github API limits to 60 requests per hour

Reply:
(Sorry, my GH tab didn't refresh and didn't see all those comments)

Reply:
May I mark it as solved? @aabadie

Reply:
ping @aabadie

Reply:
After some testing locally, I think it's worth having the token. The maximum request limit is reached quite fast (after 10 attempts or so), because of the pagination.
The current logic is fine.